atlp-rwanda/code-ninjas-be
# set up migration

### What does this PR do?
 Set up the migrations and models structure
### Description of Task to be completed?

### How should this be manually tested?
##Get started Run command bellow:
  - yarn install: to install all dependencies
  - yarn run migrate: to migrate table
  - yarn run run-seeds: to populate table with sample data
  - yarn run down: to drop table(be careful you will lose all your data)
##Test if you can access data stored in your database:
  - Create .env file
  - add database url using "DB_CONNECT" as variable
  - add database password using "DB_PASSWORD" as variable
  - Open postman
  - type in this route "localhost:3000/api/user" (you will be able to see all users in your database)

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[storyID: #181414700](https://www.pivotaltracker.com/story/show/181414700)
### Screenshots (if appropriate)
### Questions:
